internal/licenses/context_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func TestSetContextLicenseScanner(t *testing.T) {
-	scanner := testScanner(true)
+	scanner := testScanner()
```

Comment:
Scanner no longer maintains config about whether is should return content. It will always return evidence for a scanned license file.